build.sh

```diff
@@ -29,6 +29,8 @@ PRINT_USAGE() {
     echo "  -j [N], --jobs[=N]   Multicore build, allow N jobs at once"
```

Comment:
If they are two distinct changes I would opt for two separate PRs, but if there aren't any further comments at this point it would be okay to keep them together this time.